frontend/src/CourseCard.css

```diff
@@ -0,0 +1,47 @@
+.CourseCard {
+    display: flex;
+    background: #F6F8F5;
+    box-shadow: 0px 4px 25px rgba(0, 0, 0, 0.24);
+    border-radius: 15px;
+    font-family: 'Roboto', sans-serif; 
+}
+
+.CardBody {
+    align-self: center;
+    padding: 2em;
```

Comment:
I would adjust `left` padding, to reflect `66px` distance as specified on design. I guess the place where I would not use `em` as it can be unpredictable. In the end, it does render in `px`.

Reply:
added in px now. You're right, I'm never sure when to use em :/